distribution/sdk/sample-nsili-server/src/main/java/org/codice/alliance/nsili/mockserver/impl/managers/CatalogMgrImpl.java

```diff
@@ -113,7 +116,7 @@ public HitCountRequest hit_count(Query aQuery, NameValue[] properties)
             poa_.activate_object_with_id("hit_count".getBytes(Charset.forName(ENCODING)),
                     hitCountRequest);
         } catch (ServantAlreadyActive | ObjectAlreadyActive | WrongPolicy e) {
-            System.out.println("hit_count : Unable to activate hitCountRequest object.");
+            LOGGER.error("hit_count : Unable to activate hitCountRequest object.", e);
```

Comment:
Again, error seems to high to me.  What is the consequence of not getting a hit count?  I'm thinking this could be info or debug.